### PR TITLE
Added removeProfile to MagickImage.

### PR DESCRIPTION
**Pull request by RubensCordeiroo**

### CODE REVIEW

File: tests/magick-image/remove-profile.spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿// Copyright Dirk Lemstra https://github.com/dlemstra/magick-wasm.
+// Licensed under the Apache License, Version 2.0.
+
+import { TestImages } from '../test-images';
+
+describe('MagickImage#removeProfile', () => {
+    it('should remove the profile from the image', () => {
+
```
Comment (dlemstra) [inline tests/magick-image/remove-profile.spec.ts:8]: Can you remove the three empty lines after the braces? Not a big deal but it's more the style of the rest of the project.

**Review by dlemstra — COMMENTED** [review]
Thanks for helping out with this project.

**Comment by RubensCordeiroo** [comment]
Yes,  sure

File: tests/magick-image/remove-profile.spec.ts (hunk 1 above)
Comment (RubensCordeiroo) [reply tests/magick-image/remove-profile.spec.ts]: Yes, sure

**Review by dlemstra — APPROVED** [review]
Thanks for your help

**Pull request merged**